### DISCUSSION TITLE
NFT Showcase

### DISCUSSION THREAD

**Discussion by tweakch** [cardano-foundation/developer-portal#243] [General]
Can we have a discussion about the Tokens in the showcase? 

I like some projects but I struggle with approving them because apart from a neat description and/or cool idea, most of them dont provide more than a link to the homepage... and that is not enough IMHO.

Should we create an **Assets Board** where the only requirement is providing a real policy_id... and the showcase "features" different assets from that board? 

What do you think @katomm @Mercurial @rdlrt ?
What do the creators of the NFTs have to say?

**Comment by rdlrt** [comment]
My responses regarding these from previously hosted (now retired) tg group:

```
A showcase - in my eyes - could be successful (relative to "present" but gradually growing bar) projects in the ecosystem, that someone new to ecosystem can come and see what can be done

If instead - we have hundreds (the level of projects with PRs currently) of projects (be it NFTs, tokens, utility websites) ...then the showcase  instead becomes a DB and loses it's intent
```

Though it's very difficult to be able to "judge" a project, in terms of metrics - as:
- Only tx volume can be artificially made
- Intent for future uniqueness does not guarantee a successful transition (to me I cannot approve such tokens, but there are many collaborators who may disagree and approve).

Stripping off specific examples I had shared :
```
It would be nice to be able to use metrics, yes...but difficult to determine/attach value to it...In my head I think that for 'tokens' / 'NFT' the decision on whether it has unique value and widely adopted in community should be unanimous to be in showcase list...ones where we have to question should gradually become even more difficult to list, and same should go for ones already on showcase. In my mind, there are:
1. clearly agreeable show cases

And I am not really sure how I can differentiate between the two groups in my head without a human judgement involved 🤔
Would love to see any metrics (not just based on some stats like tx count that can be artificially achieved) that could potentially help us rate them.
```

Most important, it should *NOT* become something like cardano-essentials. But taking a negative stance on approving can understandably hurt a lot of enthusiasm, which is not something we want 🙂 

I suppose one way around this could be use of `Featured` flag which would allow us to show only "Featured" by default and then via expansion/seperate section list others that are not unanimously successful in individual minds?

**Comment by katomm** [comment]
I agree with both of you @tweakch @rdlrt - we need to find a good line here. Btw: everyone who cares (if you are reading this you seem to fit into this target group) is free to join the discord on https://discord.gg/xMXT523U where we are about to regroup and discuss guidelines and approaches. (link will expire in 7 days on purpose) - but most of it can and should also be easily discussed in the repo itself.

In my opinion, the goal of the showcases should not be to list EVERY project. We touched on this already in the guidelines on https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js - the main goal should be that we show people what you can do on Cardano, in different categories.

**Reply by gufmar** [reply]
Throwing in the TFC idea. It's based on CMC (coin market cap) and slightly modified to Token fee cap.

In other words: have a computed and continuosly updated top list of tokens, sorted by paid fees per epoch. 

This computed data will show some long-trending extraordinary tokens, then listed in the showcase.

**Reply by katomm** [reply]
No matter if we want to do something like that or not, I would like to know how it would look like and which tokens would be possible. Where would you draw the line? For example, in my opinion, they have made a lot of effort with Spacecoins. But it is not the most successful coin if you look at https://cardanoassets.com.

This is just a feeling but if I were to draw a line, I would draw it roughly so that Spacecoins is in it.

**Comment by katomm** [comment]
Since there is no more discussion here, I suggest we find a way to find out the 10, 15, or 20 NFT projects with the most traction within the community and then "close" the showcase submissions for NFT.

I also want to point to a great discussion around this here with a proposal from SmaugPool https://discord.com/channels/875366566944968785/875612500119408661/888322921431453706 

That said I love that we see so many NFT projects on Cardano, it's just here on the portal that it gets a little out of hand PR-wise. We don't want to map out the entire NFT space.

**Reply by rdlrt** [reply]
Fully agree

**Reply by happypus** [reply]
I'm not agree because all projects must be shown or at least 100 nfts projects should be listed on nft showcase

**Reply by rdlrt** [reply]
@happypus  -> You already have Cardano Essentials for that 👍🏻  You can list your projects there without any curation/requirements for adoption/usability/uniqueness/being live on mainnet 😉

**Comment by happypus** [comment]
so, can I be listed on showcase nft ?
________________________________

**Comment by katomm** [comment]
To bring up this discussion again, since there is still no good solution. What if we would add all the CNFT award winners of all the categories to the showcase of the developer portal? https://twitter.com/theCNFTawards/status/1485885159329128451

There, there is a similar procedure as we have already suggested: categorization and it takes the burden off us to make that decision.

**Reply by rdlrt** [reply]
I fully support this suggestion, as it gives power back to community - and folks are not disheartened by NFT PRs not getting merged to lack of unanimous decisions.

**Reply by katomm** [reply]
I wonder what happened to this, but today the website is no longer available: https://www.cnft-awards.io, EDIT: just for reference, winners are available here: https://cnftnews.com/725/

**Comment by angelmerino77** [comment]
I really think that its unfair for artist like me who just want to do what their like and make money and make more people happy with art and creation, and make it so difficult to pass that PolicyID, so, it have to be a better way to confirm our art and our NFT's

**Reply by katomm** [reply]
Would you please elaborate on that a little bit? What do you think would be fair instead?

**Reply by rdlrt** [reply]
From what I read, he's sharing opinion not about your latest suggestion (or thread), but about process that resulted in his NFT not getting merged - unfortunately in chat format.

(Apologies if I misread)

**Comment by angelmerino77** [comment]
Well, it will be fair to someone like me who send pictures of my artwork in my walls of my house in my pull request, completely original artwork and i attach the links of one .com webpage that i have, my facebook since 2012 that have all my artwork and the dates of published and its a lot of proof that its my artwork, and with all the proof my NFT was not merge, so i have my NFT siting in my ADA wallet.

**Comment by angelmerino77** [comment]
I'm not a developer and i think that should be in youtube or github maybe 2 videos or 1 clear video that show from the start to finish in how create and request step by step our policy's ID revised. The video that i found the guy said ...

project"example"
here"example"

I think was very confusing that's why I think professionals developers should make at least 2 or 3 clear videos in youtube and artist are gonna be more into ADA NFT''s. Thanks.

**Comment by rphair** [comment]
I'm just catching up on this discussion and generally I agree with the rationale to keep the showcase list relatively small in number & confined to a breadth of projects that can provide positive influence to less experienced collections & creators.  It makes sense for `cardano-essentials` to be the second tier for projects that aren't in this showcase, because it assures that _anyone_ can get an incoming link to their work & web site regardless of project status.

Also I think the PR format on GitHub provides a means of **appeal** which would make me comfortable as a reviewer in case I/we might fail to value a new project's influence or uniqueness properly.  Whether an NFT collection PR cannot get the required number of approvals for merge or is deliberately closed, the collection owners, artists & advocates might still post in the PR with additional points: hopefully not to harangue or influence a decision with trivial "votes" but to form the basis of a rational appeal over time.

I would guess some of this is already being done, but I've still only read very few PR's so I don't know if any editorial decisions were ever reversed by popular demand.  In any case, as long as the possibility of "appeal" exists and is honoured by the reviewers, then I would feel most comfortable having a high barrier of entry for new projects to include in the NFT showcase.

**Comment by angelmerino77** [comment]


**Reply by katomm** [reply]
> Dont worry about it, i see you soon in the metaverse and im not gonna give a fuck any merge or any policy id to sell my NFT sooo i dont think i need anything of you to make money and be active inside and outside of the metaverse. Soooo get the fuck out of here. 
El miércoles, 13 de julio de 2022, 10:50:42 p.m.

@angelmerino77 unfortunately this is exactly the mindset that leads to this discussion. It is not the purpose of the portal that people promote/sell NFT but I wish you well.

**Comment by angelmerino77** [comment]
Let me make you wiser my friend, i send pictures of my Cardano NFT my art in my house in all my social media, and sorry but my art its pretty good ANYWAYS ho are a single to judge creativity, Blockchain its for everybody no one should stop creativity and art but thank God that there are many blockchains. Jejeje and God need to add in the Bible - and the men never get to trust another men and they create the blockchain.
    El jueves, 14 de julio de 2022, 12:54:43 a.m. AST, Tommy Kammerer ***@***.***> escribió:  
 
 


@angelmerino77 unfortunately this is exactly the mindset that leads to this discussion. It is not the purpose of the portal that people promote/sell NFT but I wish you well.

**Reply by katomm** [reply]
@angelmerino77 The fact that I give my opinion here on why this discussion is essential does not stop you from using Cardano. I appreciate anyone who builds on Cardano.

**Comment by rphair** [comment]
Maybe regarding recent point https://github.com/cardano-foundation/developer-portal/pull/670#issuecomment-1259545906 the best way to move forward is to demand some **utility** (_currently implemented_, not just planned) from the NFTs & project.  Even if demanding only game-based utility it would serve as a weeder for the geometrically increasing number of submissions.

And if a well-adopted NFT project ends up being disqualified on that basis, the submitter might offer further proof (as I suggested in https://github.com/cardano-foundation/developer-portal/discussions/243#discussioncomment-3142590) that their project, which may still be lacking utility, represents some other kind of value to the ecosystem.

**Comment by katomm** [comment]
Another try to bring up this discussion with a rather pragmatic proposal, listing X top projects listed on https://opencnft.io/?t=all (by all-time volume) I am aware that there are some possibilities for manipulation, but the 10, 15 top projects there are certainly known, have their communities, and we would have a clear metric for the showcase. (in some way what was discussed in the very first days but back then there were no sites available for this data)

**Reply by rphair** [reply]
Sounds good to me.  I had been thinking demonstrable [utility](https://github.com/cardano-foundation/developer-portal/discussions/243#discussioncomment-3743949) could be an independent qualifier for entry... but on further consideration I think utility will be reflected in NFT collections' ranking on OpenCNFT (the more & better utility an NFT has, the more likely it will be to appear in the top 10 or 15 items).

**Reply by rphair** [reply]
We could be covered either way by also allowing NFT providers to make a case for their projects whose NFTs have some utility but might not be ranked.  This would avoid our refusing projects that demonstrate some technical advance for Cardano simply because their projects aren't commercially popular. 🤔

**Comment by katomm** [comment]
NFT update proposal that considered everything that was discussed: #1012
